airflow/providers/opsgenie/operators/opsgenie_alert.py

```diff
@@ -92,7 +93,7 @@ def __init__(self,
         self.alias = alias
         self.description = description
         self.responders = responders
-        self.visibleTo = visibleTo
+        self.visibleTo = visibleTo  # pylint: disable=invalid-name
```

Comment:
I would be in favor of renaming + note in updating.md

Reply:
refactor